### ISSUE TITLE
Error: Could not find a package configuration file provided by "Qt5Widgets"

### ISSUE THREAD

**Issue by trusktr** [equalsraf/neovim-qt#26]
I'm getting this error during build. I believe I have Qt5 installed since Qt Creator works. The error:

```
CMake Error at CMakeLists.txt:32 (find_package):
  By not providing "FindQt5Widgets.cmake" in CMAKE_MODULE_PATH this project
  has asked CMake to find a package configuration file provided by
  "Qt5Widgets", but CMake did not find one.

  Could not find a package configuration file provided by "Qt5Widgets" with
  any of the following names:

    Qt5WidgetsConfig.cmake
    qt5widgets-config.cmake

  Add the installation prefix of "Qt5Widgets" to CMAKE_PREFIX_PATH or set
  "Qt5Widgets_DIR" to a directory containing one of the above files.  If
  "Qt5Widgets" provides a separate development package or SDK, be sure it has
  been installed.
```

I'll try updating it...

**Comment by equalsraf** [comment]
Cheers,  can you provide some details about your system? e.g. if this is Windows and you are building from the console, the Qt environment needs to be initialized (see #20) by calling qtenv2.bat.

> I'll try updating it...

We should be able to build with Qt 5.2 or later.

**Comment by trusktr** [comment]
Hey @equalsraf, I'm in OS X 10.10.3 (14D136). Do I need to put Qt bins in a env variable?

**Comment by equalsraf** [comment]
Not sure about Mac (are you using homebrew?), CMake needs to be able to find the cmake scripts provided by Qt i.e.CMAKE_PREFIX_PATH should point to the Qt prefix - for example this [comment](https://github.com/Cockatrice/Cockatrice/issues/205#issuecomment-48705334) or maybe something like [this](https://github.com/Cockatrice/Cockatrice/wiki/Installing-Cockatrice#compiling-with-homebrew) for homebrew

**Comment by trusktr** [comment]
@equalsraf Idea: Can you make this a Qt Creator project? Then we won't have to worry about compile steps on different OSes.

**Comment by equalsraf** [comment]
@trusktr that should already work - you should be able to open the CMakeLists.txt from Qt Creator (Open Project). You may need to provide the CMake binary location.

**Comment by trusktr** [comment]
Interesting. I never used Qt Creator like that before. I ran cmake with the Unix Generator and no arguments. Then that took me into Qt where I pressed the green Play button to build, and it opened this window:

<img width="828" alt="screen shot 2015-07-07 at 5 00 52 pm" src="https://cloud.githubusercontent.com/assets/297678/8560146/d8528a5c-24c9-11e5-8d50-5156c0b93ce4.png">

I was hoping to see a NeoVim intro screen in there and begin editing text. Do I have to make a neovim instane in terminal?
